launch-checklist.md

```diff
@@ -1 +1,20 @@
+# Replicache Launch Checklist
 
+Before you launch with Replicache in your product, it's a good idea to double-check that you have correctly covered all the small details of integration. This list can help you determine if you might have missed a detail, or deferred and then forgotten about it.
+
+- **JS SDK**
+  - if you wish to version the schema of the mutations in `Push` and the `clientView` returned in `Pull`, version information is provided in `ReplicacheOptions.schemaVersion`
```

Comment:
Grammatical nit, but this sounds informational and not like a task the dev should do. Rephrase like: "consider adding a `schemaVersion` field to `ReplicacheOptions` proactively so that you don't have to have a special case for "no schemaVersion" later.